unity-renderer/Assets/DCLPlugins/ECS7/TestsUtils/ECS7TestScene.cs

```diff
@@ -29,6 +29,13 @@ IDCLEntity IParcelScene.CreateEntity(long id)
         return CreateEntity(id);
     }
 
+    public IDCLEntity GetEntityById(long entityId)
+    {
+        if (!entities.TryGetValue(entityId, out IDCLEntity entity)) { return null; }
+
+        return entity;
+    }
```

Comment:
why do we need this method implementation?

Reply:
Why wouldn't we? isn't the TestEntity simulating the real `DecentralandEntity`? `DecentralandEntity` has that, so if we want to be able to run test stuff with test entities instead of real entities, better to have that.

Reply:
Done